stub-idp/src/integration-test/java/uk/gov/ida/apprule/ServiceListServiceIntegrationTest.java

```diff
@@ -0,0 +1,98 @@
+
+import javax.inject.Inject;
+import java.util.Optional;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class ServiceListServiceIntegrationTest extends IntegrationTestHelper {
+
+    private final String MOCK_RESPONSE = "[ { \"name\": \"Service 1\", \"loa\": \"LEVEL_2\", \"entityId\": \"service-1-entityId\", \"serviceCategory\": \"Service Category A\" }, { \"name\": \"Service 2\", \"loa\": \"LEVEL_2\", \"entityId\": \"service-2-entityId\", \"serviceCategory\": \"Service Category B\" }, { \"name\": \"Service 3\", \"loa\": \"LEVEL_1\", \"entityId\": \"service-3-entityId\", \"serviceCategory\": \"Service Category A\" } ]";
+    private static final int WIREMOCK_PORT = 58888;
+
+    @Rule
+    public final WireMockRule wireMockRule = new WireMockRule(WIREMOCK_PORT);
+
+    @ClassRule
+    public static final StubIdpAppRule applicationRule = new StubIdpAppRule(
+        ConfigOverride.config("singleIdpJourney.enabled", "true"),
+        ConfigOverride.config("singleIdpJourney.serviceListUri", String.format("http://localhost:%d/get-available-services", WIREMOCK_PORT))
+    );
+
+    private JsonClient jsonClient;
+
+    @Before
+    public void setUp() {
+        JerseyClientConfiguration clientConfig = new JerseyClientConfiguration();
+        clientConfig.setRetries(3);
+        clientConfig.setTimeout(Duration.milliseconds(1000));
+        clientConfig.setConnectionTimeout(Duration.milliseconds(1000));
+
+        ClientProvider provider = new ClientProvider(
+                applicationRule.getEnvironment(),
+                clientConfig,
+                true,
+                "socketTimeoutRetryWithBackoffTests_thirdCallSucceedsClient"
+        );
+
+        jsonClient = new JsonClient(new ErrorHandlingClient(provider.get()), new JsonResponseProcessor(applicationRule.getObjectMapper()));
+    }
+
+    @Test
+    @Inject
+    public void getListOfServices_SuccessfulResponseFromHub() throws FeatureNotEnabledException {
+        final String scenarioName = "Successful Read Scenario";
+        final String resourcePath = "/get-available-services";
+
+        stubFor(get(urlEqualTo(resourcePath)).inScenario(scenarioName)
+                .withHeader("Accept", equalTo("application/json"))
+                .willReturn(aResponse()
+                        .withStatus(200)
+                        .withHeader("Content-Type", "application/json")
+                        .withBody(MOCK_RESPONSE)
+                )
+        );
+
+        SingleIdpConfiguration configuration = applicationRule.getConfiguration().getSingleIdpJourneyConfiguration();
+
+        ServiceListService listService = new ServiceListService(configuration, jsonClient);
+
+        Optional<Service[]> services = listService.getServices();
+
+        assertTrue(services.isPresent());
+        assertEquals(services.get().length, 3);
```

Comment:
We should use the AssertJ matchers e.g. assertThat to keep the test consistent with the other tests